testkit/src/test/scala/org/scanamo/fixtures/fixtures.scala

```diff
@@ -6,13 +6,15 @@ case class City(name: String, country: String)
 case class Doctor(actor: String, regeneration: Int)
 case class Engine(name: String, number: Int)
 case class Factory(id: Int, name: String)
-case class Farm(asyncAnimals: List[String])
+case class Farm(animals: List[String], hectares: Int)
 case class Farmer(name: String, age: Long, farm: Farm)
 case class Forecast(location: String, weather: String, equipment: Option[String])
-case class Gremlin(number: Int, wet: Boolean)
+case class Gremlin(number: Int, wet: Boolean, friendly: Boolean)
```

Comment:
I can see that `hectares` & `friendly` have been added as constructor parameters to `Farm` & `Gremlin` here, and this has led to a fair few bits of diff in the tests. Just checking, was this necessary to test some aspect of Scala 3 support, or could it be left out of this PR?

Reply:
Ah I see, it came from `TableTest`:

https://github.com/scanamo/scanamo/blob/36dd69ff8b11a7392c5e1e626e7c06aa1272d82a/scanamo/src/test/scala-2.x/org/scanamo/TableTest.scala#L21-L27